README.md

```diff
@@ -173,6 +173,12 @@ A good place to start is with an issue that has the label [good first issue](htt
 4. Push to the Branch (`git push origin f/AmazingFeature`)
 5. Open a Pull Request
 
+### Adding new API
+
+- Add into `/api/db` and `/api/graphql` your definitions.
+
+- Then import them in `/api/index/`
```

Comment:
Maybe you can explain how to import them into `api/index/ ` ?

Reply:
remove as per our discussion.